tests/subaccounts/createSubAccount.test.js

```diff
@@ -14,19 +14,19 @@ test('name is generated in subaccount create', async () => {
     let subaccount = await client.subaccounts.create()
 
     expect(subaccount.name).toBeTruthy()
+    expect(subaccount.email).toBeFalsy()
 })
 
-test('subaccount create with email parameter', async () => {
+test('subaccount create with email parameter does nothing', async () => {
```

Comment:
We don't need the email field on subaccounts anymore in our clients; We just kept it at the server side to avoid sending bad requests suddenly.